pkg/Microsoft.ML.CodeGen/Microsoft.ML.CodeGen.nupkgproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET Code Generator</PackageDescription>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="../Microsoft.ML.AutoML/Microsoft.ML.AutoML.nupkgproj" />
```

Comment:
The `.nupkgproj` file needs to have the same `PackageReference`s as the corresponding .csproj.